argostime/crawler/shop_info.py

```diff
@@ -1,101 +0,0 @@
-#!/usr/bin/env python3
-"""
-    crawler/shop_info.py
-
-    Contains information about supported webshops
-
-    Copyright (c) 2022 Martijn <martijn [at] mrtijn.nl>
-
-    This file is part of Argostimè.
-
-    Argostimè is free software: you can redistribute it and/or modify
-    it under the terms of the GNU Affero General Public License as published by
-    the Free Software Foundation, either version 3 of the License, or
-    (at your option) any later version.
-
-    Argostimè is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-    GNU Affero General Public License for more details.
-
-    You should have received a copy of the GNU Affero General Public License
-    along with Argostimè. If not, see <https://www.gnu.org/licenses/>.
-"""
-
-shops_info = {
```

Comment:
where is shops_info defined if it's removed here?

Reply:
There is `enabled_shops` in `crawl_utils` that contain `ShopDict`.

`shop_info` was no longer needed after my changes. Also check the changes in argostime/products.py to see how you can use `enabled_shops`.